tests/test_s3.py

```diff
@@ -43,6 +44,16 @@ def setUp(self):
         # Enable S3 mocking of tests.
         self.mock_s3.start()
 
+        self.endpoint = None
+
+        if "S3_ENDPOINT_URL" in os.environ:
+            self.endpoint = os.environ["S3_ENDPOINT_URL"]
+            os.environ["S3_ENDPOINT_URL"] = ""
+
+        for k, v in os.environ.items():
```

Comment:
You need to remove these debugging lines

Reply:
whoops! I'll get on that